WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/PrepublishingHomeViewModel.kt

```diff
@@ -256,4 +238,17 @@ class PrepublishingHomeViewModel @Inject constructor(
     private fun onActionClicked(actionType: ActionType) {
         _onActionClicked.postValue(Event(actionType))
     }
+
+    fun updateJetpackSocialState(
+        state: EditorJetpackSocialViewModel.JetpackSocialUiState?
+    ) {
+        val newState = state?.let {
+            SocialUiState.Visible(
+                it,
+                onItemClicked = { Log.d("thomashorta", "hey there!") }
```

Comment:
❓ Just wanted to confirm that we should merge this because it's going to be addressed in another PR.

Reply:
I think it's fine merging it since this all is behind a Feature Flag anyway, and the PR removing it is already up for review.